### DISCUSSION TITLE
antD1.7.8表格头与表格体在谷歌浏览器上错位

### DISCUSSION THREAD

**Discussion by XieGeorge** [vueComponent/ant-design-vue#7324] [Q&A]
antD1.7.8表格头与表格体在谷歌浏览器上表现错位
为啥谷歌浏览器上改了滚动条样式后会错位呀？
图2那种固定一列的就不会
![3E3CF5C1-5C99-4f91-A6E5-E2BB0D791AB0](https://github.com/vueComponent/ant-design-vue/assets/86343409/d44f287a-69b2-4cb9-ae85-271e8c0c6e44)
![DF544C01-887B-4318-8B66-9298619BE4E1](https://github.com/vueComponent/ant-design-vue/assets/86343409/f41e5a7a-a4b9-457a-a94e-fb6d03117a5e)

**Comment by pavitra-infocusp** [comment]
I can confirm this
